### DISCUSSION TITLE
game crashed when accidentally hitting f11

### DISCUSSION THREAD

**Discussion by T1hy** [ppy/osu#18591] [Q&A] [answered]
### Type
Crash on start up
### Bug description
I hit f11 and the game switched to window mode, when i hit f11 again the game crashed. It didnt give me any crash message. I tried reinstalling osu lazer but the problem keeps happening, I even deleted normal osu.
When i open the installer it shows me the osu logo for some time and then opens to the game screen but closes after two seconds. Then when i try to open the game it just crashes after a black screen. It's the latest version available to download on this github.
I found this same issue happened to another user but the discussion was never resolved.
Edit: I realized that when the game is alt tabbed while it opens, it start up normally, but when i clic on it, it instantly closes.
### Version
2022.605.1 (latest)
### Logs
[performance.log](https://github.com/ppy/osu/files/8844390/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8844392/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8844393/updater.log)
[database.log](https://github.com/ppy/osu/files/8844394/database.log)
[network.log](https://github.com/ppy/osu/files/8844395/network.log)

**Comment by bdach** [comment, marked as answer]
please update graphics card drivers and/or disable shadowplay.
